Public/Start-Polaris.ps1

```diff
@@ -0,0 +1,46 @@
+<#
+.SYNOPSIS
+    Start Polaris web server.
+.DESCRIPTION
+    Start Polaris web server.
+.PARAMETER Port
+    Port number to listen on.
+    Defaults to 8080.
+.PARAMETER MinRunspaces
+    Minimum number of PowerShell runspaces for web server to use.
+    Defaults to 1.
+.PARAMETER MaxRunspaces
+    Maximum number of PowerShell runspaces for web server to use.
+    Defaults to 1.
+.PARAMETER UseJsonBodyParserMiddleware
+    When present, JSONBodyParser middleware will be created, if needed.
+.EXAMPLE
+    Start-Polaris
+.EXAMPLE
+    Start-Polaris -Port 8081 -MinRunspaces 2 -MaxRunspaces 10 -UseJsonBodyParserMiddleware
+#>
+function Start-Polaris {
```

Comment:
Nice to have: This function should take in a Polaris class object with a default value of $script:Polaris so that people can have multiple instances running in the same session.